RF24Ethernet.h

```diff
@@ -206,6 +229,8 @@ class RF24EthernetClass
 
 extern RF24EthernetClass RF24Ethernet;
 
+typedef RF24EthernetClass RF52EthernetClass;
+
 #endif // RF24Ethernet_h
```

Comment:
The extra indent of pre-processor cmds is due to the fact that clang-format expects this `#endif` to be the absolute last statement in the header file.

Moving the `#endif` to EoF shouldn't have any consequences.

Reply:
Updated and note added to readme